pyecharts/charts/basic_charts/sunburst.py

```diff
@@ -0,0 +1,50 @@
+from ... import options as opts
+from ...charts.chart import Chart
+from ...commons.types import JSFunc, Optional, Sequence, Union
+from ...globals import ChartType
+
+
+class Sunburst(Chart):
+    """
+    <<< 旭日图 >>>
```

Comment:
English doc string please! https://github.com/pyecharts/pyecharts/issues/965

Reply:
👌

Reply:
Sunburst Chart is composed of multiple pie charts. From the view of data structure, inner rings are the parent nodes of outer rings. Therefore, it can show the partial-overall relationship as Pie charts, and also level relation as Treemap charts.